indexing-service/src/test/java/io/druid/indexing/overlord/RemoteTaskRunnerTest.java

```diff
@@ -335,7 +338,7 @@ public void testRunWithTaskComplete() throws Exception
 
     ListenableFuture<TaskStatus> future = remoteTaskRunner.run(task);
 
-    TaskStatus status = future.get();
+    TaskStatus status = future.get(TIMEOUT_SECONDS, TimeUnit.SECONDS);
```

Comment:
going forward, let's always put a 60s timeout on the test method, and not have arbitrary timeouts in-between.